main.go

```diff
@@ -572,44 +563,44 @@ func SetupDir(s *State) bool {
 	return true
 }
 
-func ProcessDnsEntry(s *State, word string, resultChan chan<- Result) {
-	subdomain := word + "." + s.Url
+func processDNSEntry(s *state, word string, resultChan chan<- result) {
+	subdomain := word + "." + s.URL
 	ips, err := net.LookupHost(subdomain)
 
 	if err == nil {
-		if !s.IsWildcard || !s.WildcardIps.ContainsAny(ips) {
-			result := Result{
+		if !s.IsWildcard || !s.WildcardIps.containsAny(ips) {
+			r := result{
```

Comment:
use a short name for the local variable to keep from having too many similar names